packages/web-components/src/components/pictogram-item/defs.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Pictogram colors.
+ */
+export enum COLOR_OPTIONS {
+  /**
+   * Defaults to black.
```

Comment:
This is a bit minor, but I'm not sure if "black" is a good label, I think we can keep to `Gray 100` or `g10` to align with the Carbon theme. Anywho, I'll defer to @oliviaflory or @RichKummer on this.

Reply:
I double checked and it looks like the Pictogram is using `text-01` to maintain readability across the themes, so we should call the default option `text-01` to align with that - otherwise "Black" would not make sense when it swaps to a lighter version on Gray 90/100.

Reply:
Sorry, disregard the other comment - 

I don't think the "Black" works since the color token swaps to a lighter version on Gray 90/100. Naming the default option "Inverse" would clarify that and make it readable. We should probably change "Blue 50" to "Blue" since the the token used is slightly different based on Theme as well.

Reply:
@proeung @RichKummer I changed the description to "text color" since "Inverse" doesn't make sense to me as the default on the lighter schemes. What do you think?

I removed the "50" from blue.

<img width="821" alt="Screen Shot 2022-04-07 at 10 03 01 PM" src="https://user-images.githubusercontent.com/384598/162368276-1067e913-a676-4c9c-8e5e-734a1090d03e.png">

<img width="821" alt="Screen Shot 2022-04-07 at 10 03 52 PM" src="https://user-images.githubusercontent.com/384598/162368462-5636fa99-1e95-42d8-ac01-e647f7ea7645.png">

Reply:
@mwanberg I noticed the WC build failed on the last commit, does this need to be rerun/merged?

Reply:
@RichKummer I just merged in the latest from `main`, which has kicked off another build. Let's hope it finishes this time. 🤞🏻